com.unity.render-pipelines.universal/Runtime/UniversalRenderer.cs

```diff
@@ -463,12 +528,21 @@ public override void Setup(ScriptableRenderContext context, ref RenderingData re
                     // Deferred renderer does not require a depth-prepass to generate samplable depth texture.
                     if (this.actualRenderingMode != RenderingMode.Deferred)
                     {
-                        m_DepthPrepass.Setup(cameraTargetDescriptor, m_DepthTexture);
+                        m_DepthPrepass.Setup(cameraTargetDescriptor, m_DepthTexture, m_ActiveCameraDepthAttachment, useDepthPriming);
                         EnqueuePass(m_DepthPrepass);
                     }
                 }
             }
 
+            // Depth priming requires a manual resolve of MSAA depth right after the depth prepass.
+            if (useDepthPriming)
+            {
+                m_PrimedDepthCopyPass.Setup(m_ActiveCameraDepthAttachment, m_DepthTexture);
+                m_PrimedDepthCopyPass.AllocateRT = false;
+
+                EnqueuePass(m_PrimedDepthCopyPass);
+            }
```

Comment:
@manuele-bonanno for awareness as it seems this could be optimized in a follow up PR with your work?

Reply:
yes this could be removed on platforms with depth resolve support (once my PR lands)

Reply:
Something that might be an issue here.
If you look at CanCopyDepth(), there is this TODO comment:
// TODO:  We don't have support to highp Texture2DMS currently and this breaks depth precision.
 
 because of this, using MSAA'd textures as source of the copy is currently disabled by default.
 So if that is still the case (unless that comment is outdated), aren't we breaking potentially depth precision here by copying a multisampled surface?

Reply:
@pbbastian confirmed that we now have highp support for Texture2DMS so all good :)